spec/datagrid/columns_spec.rb

```diff
@@ -98,6 +98,30 @@ class Report27 < Datagrid::Base
           expect(Report27.new.header.first).to eq("Nombre")
         end
       end
+
+      it "uses configured default header" do
+        grid = test_grid do
+          self.default_column_options = ->(column) { { header: -> { I18n.t(column.name, scope: "other.location") } } }
+
+          scope { Entry }
+          column(:name)
+        end
+
+        store_translations(:en, other: { location: { name: "Nosaukums" } }) do
```

Comment:
I didn't quite get - where is this function defined?

Reply:
`spec/support/i18n_helpers.rb` - saw it used in other tests, assumed it was the way to handle translations in the test suite.